modules/couchbase/src/test/java/org/testcontainers/couchbase/CouchbaseContainerTest.java

```diff
@@ -12,4 +12,11 @@ public void shouldUseCorrectDockerImage() {
         Assert.assertEquals(CouchbaseContainer.DOCKER_IMAGE_NAME + CouchbaseContainer.VERSION,
             couchbaseContainer.getDockerImageName());
     }
+
+    @Test
+    public void shouldStopWithoutThrowingException() {
```

Comment:
since this test does not test anything automatically, I would remove it, just to not add time to the Testcontainers test suite

Reply:
I'm fine with removing it, as long as it's understood that this test fails prior the main code change in this PR.

Reply:
ah, ok, I thought that it is only about the logging, but if it fails with current master - let's keep it 👍